File: web/src/models/DTOs/ConsultaBoletoCaixa/ServicoSaidaDTO.ts
import { ConsultaBoletoDTO } from "./ConsultaBoletoDTO";
import { ControleNegocialDTO } from "./ControleNegocialDTO";
import { HeaderDTO } from "./HeaderDTO";

export interface ServicoSaidaDTO {
  header: HeaderDTO;
  codRetorno: string;
  origemRetorno: string;
  msgRetorno: string;
  dados: {
    controleNegocial: ControleNegocialDTO;
    consultaBoleto: ConsultaBoletoDTO;
  };
}

File: webservices/src/main/java/fieg/modulos/boleto/dto/ConsultaBoletoCaixa/ServicoSaidaDTO.java
package fieg.modulos.boleto.dto.ConsultaBoletoCaixa;

import fieg.externos.caixaboletoservice.consulta.dto.DadosDTO;
import fieg.externos.caixaboletoservice.consulta.dto.HeaderDTO;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class ServicoSaidaDTO {

    private HeaderDTO header;
    private String codRetorno;
    private String origemRetorno;
    private String msgRetorno;
    private DadosDTO dados;
}
